proto/common.proto

```diff
@@ -417,7 +417,7 @@ message RaidBuffs {
 	TristateEffect strength_of_earth_totem = 14; // none, normal, 15% improved
 	
 	// 20% Melee Haste
-	TristateEffect icy_talons = 15;
+	bool icy_talons = 15;
```

Comment:
Ahh, I thought icy talons was like WF which can be 16% or 20%. I will fix this in my next PR

Reply:
ah nvermind, you already fixed it :)